ArcadiaParties.API/Startup.cs

```diff
@@ -56,6 +60,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
+            app.UseAuthentication();
+
+            app.UseCreateClaims(); //Custom middleware
```

Comment:
Comments should describe not **what is done**, but **why it is done that way**. That's why this comment is not needed.

Reply:
Comment has deleted

Reply:
Comment has deleted